decred_wallet/Features/Security/RequestPinViewController.swift

```diff
@@ -76,15 +77,26 @@ class RequestPinViewController: SecurityBaseViewController {
             self.pinInputView.clear()
             self.headerText.text = "Confirm \(self.securityFor) PIN"
             self.prgsPinStrength.progress = 0
+            
+            // We are confirming pin, we hide the pin strength meter
+            self.pinStrengthLabel.isHidden = true
+            self.prgsPinStrength.isHidden = true
         }
         else if requestPinConfirmation && pinToConfirm != pinInputView.pin {
             self.pinToConfirm = ""
             self.headerText.text = "PINs did not match. Try again"
+            
+            // Reset the input
             DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
                 self.headerText.text = "Create \(self.securityFor) PIN"
                 self.pinInputView.clear()
                 self.prgsPinStrength.progress = 0
+                
+                // We're re-requesting input, show the strength meter
+                self.pinStrengthLabel.isHidden = false
+                self.prgsPinStrength.isHidden = false
             }
+            
```

Comment:
Don't see a need for this empty line space.

Reply:
Fixed in commit d097f900b6d316d6272559d520696d0fec66710a